app/models/order.rb

```diff
@@ -12,6 +13,8 @@ class Order < ApplicationRecord
   validates :name, :address, :email, presence: true
   validates :pay_type, inclusion: pay_types.keys
 
+  scope :by_date, ->(from = Date.today, to = (Date.today + 1)) { Order.where("created_at >= ?", from).where("created_at < ?", to) }
```

Comment:
you can use Date.tomorrow also

Reply:
what is the difference between `Date.today + 1` and `Date.today + 1.day`?

Reply:
Made changes

Reply:
Please answer the questions asked